maat-orchestration/src/main/java/uk/gov/justice/laa/crime/orchestration/service/orchestration/MeansAssessmentOrchestrationService.java

```diff
@@ -64,11 +71,12 @@ public ApplicationDTO create(WorkflowRequest request) {
     }
 
     public ApplicationDTO update(WorkflowRequest request) {
-
         ApplicationDTO application = request.getApplicationDTO();
         try {
             Long repId = application.getRepId();
             log.debug("Updating Means assessment for applicationId = " + repId);
+
+            preProcessRequest(request, Action.UPDATE_ASSESSMENT);
```

Comment:
And again here, Is this the correct action? There wasn't anything obvious in the stored procedure.

Reply:
this is the correct action, the stored procedures have been retired. Please refer https://github.com/ministryofjustice/laa-maat-application/pull/504/files